pkg/container/device.go

```diff
@@ -77,6 +79,8 @@ func (c *container) specAddHostDevicesIfPrivileged(privilegedWithoutHostDevices
 func (c *container) specAddContainerConfigDevices() error {
 	sp := c.Spec().Config
 
+	disableDeviceOwnershipFromSecurityContext := checkDisableDeviceOwnershipFromSecurityContextAnnotation(c.Config().Annotations)
```

Comment:
if we go with the approach above, then we only have to check if the useDeviceOwnershipFromSecurityContext is in the annotations still here. The filtering of the annotations has already happened, so we know if the annotation exists, we're allowed to use it